apps/cookbook/src/app/showcase/colors-showcase/colors-showcase.component.scss

```diff
@@ -1,14 +1,14 @@
 @use '@kirbydesign/core/src/scss/utils';
 
 @each $color-name, $color-value in utils.get-all-colors() {
-  .#{$color-name} {
+  .color-sample.#{$color-name} {
```

Comment:
The "color-sample-example" section should have the `color-sample` class added to preserve the functionality of the section "_Example use (Click on a color below to change it)_"

Reply:
Right now it is just blank in stead of this:
<img width="163" alt="image" src="https://github.com/kirbydesign/designsystem/assets/42470636/dc174f7e-0129-4d3c-b16f-4ce6162709df">